barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -519,10 +508,8 @@ void client_ivc_prove_output_all(const std::string& bytecodePath,
 
     // Write the proof and verification keys into the working directory in  'binary' format (in practice it seems this
     // directory is passed by bb.js)
-    std::string vkPath = outputPath + "/mega_vk"; // the vk of the last circuit in the stack
+    std::string vkPath = outputPath + "/client_ivc_vk"; // the vk of the last circuit in the stack
```

Comment:
stale comment

Reply:
Made the change in my API PR that builds off this one